docs/extensions/reva/testing.md

```diff
@@ -62,16 +62,16 @@ export REVA_DATAGATEWAY_URL=http://localhost:9140/data # needed because the prox
 
 Then you need to start the ocis-reva services
 ```
-bin/ocis-reva frontend & \
-bin/ocis-reva gateway & \
-bin/ocis-reva auth-basic & \
-bin/ocis-reva auth-bearer & \
-bin/ocis-reva sharing & \
-bin/ocis-reva storage-home & \
-bin/ocis-reva storage-home-data & \
-bin/ocis-reva storage-oc & \
-bin/ocis-reva storage-oc-data & \
-bin/ocis-reva users &
+bin/reva frontend & \
+bin/reva gateway & \
+bin/reva auth-basic & \
+bin/reva auth-bearer & \
+bin/reva sharing & \
+bin/reva storage-home & \
+bin/reva storage-home-data & \
+bin/reva storage-oc & \
+bin/reva storage-oc-data & \
+bin/reva users &
```

Comment:
afaict we are going to have the non storage related services in a different repo.
- the `users` will  be implemented by a micro verison, acting as a proxy to the accounts service
- the frontend will only contain the data gateway, ocs and ocdav will move to ocis
- auth basic and auth bearer are already handled by the proxy ...

which leaves the gateway, sharing and the storage / data providers

having a `storage` command for those is ok ... or we split this into dedicated services ond get rid of the sub commands ... :thinking: